### DISCUSSION TITLE
Brainstorming research questions

### DISCUSSION THREAD

**Discussion by Jdlopez0108** [Open-Research-Program/HCOE#13] [General]
Asking Research Questions

This idea is from a [podcast](https://youtu.be/uap8r40LiWA?t=2800) I listened to with Dr. Anupam Jena, professor of health care policy at Harvard Medical School and physician at Massachusetts General Hospital. He talks about developing your ability to come up with good ideas by coming up with 10 random research questions per week. The example he used was, "Is there higher mortality from heart attacks during marathons because of street closures?". Don't do a literature review or look into it much. Just practice asking questions! I thought this was an awesome Idea and we could start to compile some of those research questions here.

Here are a few of my own to start.

- Does mortality for delivery drivers increase with increased use of Amazon/Grubhub/etc?
- Does use of sunscreen decrease the incidence of skin cancer?
- Do athletes with asthma experience increased cardiorespiratory benefits from training.?
- Do tight shoes cause bunions?
- Do you absorb vitamin D from the sun more or less in certain areas?
- Are people with dirty bathrooms at an increased risk of of bacterial infection?
- Does socioeconomic status affect motor and speech development?
- How does myofascial release compare to Tylenol for lower back pain.

I'm going to continue to throw questions in here, please contribute some of your own!

**Comment by Jdlopez0108** [comment]
[Here](https://www.youtube.com/watch?v=paQ2AnTW47g) is a great Ted Talk by Dr. Jena that explains the marathon question, as well as the concept of "idea sessions".

**Comment by bguill3** [comment]
1. Does eating oatmeal every morning improve cardiovascular health? 
2. What is the long term effect of running long distances on knee joint health? 
3. Does the number of networking contacts you have in healthcare prior to applying, improve your acceptance rate into medical school? 
4. Do children who experienced education solely online for 2+ years have increased anxiety regarding social interactions? 
5. Does having a job in high school improve spending habits later on in life?

**Comment by ADParedes** [comment]
Love this idea. In practice, it was harder than I thought, but here's a few pie in the sky ones to start with 

- how do different languages shape your brain development, especially wrt behavior characteristics from childhood to adulthood?  Is there one language that promotes a person to be more empathetic? Is there one language that promotes more assertiveness?  
- Is happiness truly measurable and comparable?  Can we create a scale / score that is universal that is not just wealth related for defining success in society?
- Is education prior to college antiquated? What are the alternative models popularized (like school on the cloud)  and how can it start to be advocated for changing the education system to encourage a healthy love for the work you do as opposed to just societal status?
- What are the incidences of misdiagnosis for ADD? Is ADD a fictitious epidemic? Are we misdirecting children who have a large capacity for creativity?
- How can cognitative fatigue be measured, and can a tool be implemented to show us how it can be leveled up?  
- Can mental health and body health correlation be mapped more accurately and quantitatively?

**Comment by jasson907** [comment]
1. Do students who own a dog in medical school experience more or less anxiety during school? 
2. Do healthcare workers experience more or less health issues compared to non-healthcare workers? 
3. Are people who had chores as kids, more driven? 
4. Does eating an apple a day actually keep the doctor away?

**Comment by ADParedes** [comment]
- does listening to a certain type of music help you work?  
- do personalities map to a neurobiological predisposition?
- does a patient mentality really affect surgical survival outcomes? If so why, what science does that fit into?  
- what is going on with neurolink by Elon Musk: " The Neuralink app would allow you to control your iOS device, keyboard and mouse directly with the activity of your brain, just by thinking about it."

**Comment by Jdlopez0108** [comment]
- Does Marijuana/Vape/Cigarette smoke cause direct lung damage distinct from the effect of toxins and carcinogens? 
- Do people in long-term relationships accumulate more wealth? 
- Are people in relationships more organized?
- Do states with harsh winters have higher incidence of Depression?
- Does hospital food lower quality of life for patients?
- Does alcohol have effects on blood sugar independent of sugar content?
- Do continuous glucose monitors improve outcomes as placebo alone? 
- Can childhood behavioral issues be improved by identifying and fulfilling sensory needs?
- Can childhood behavioral issues be improved by improving diet quality?
- Is creativity quantifiable? if so, does it decrease with age? 
- Are attention spans decreasing? 
- What actionable protocols can increase attention span?
- Does poor posture decrease attention and vice versa?
- Are electric toothbrushes better than regular?
- Is "Dad Bod" a result of real physiologic processes?
- Do tight clothes limit mobility and/or cause downstream musculoskeletal imbalances?
- Do people have increased allergies on days of increased traffic and/or during airplane and boat shows?
- Are there differences between oral/IV vitamins distinct from dose?
- Do medical treatments become adopted with greater ease or with less evidence as a result of positive social media and news outlet publicity?
- Do people with specific ethnic backgrounds exhibit increased/decreased tolerance to foods based on the foods available in the areas that their ancestors originated from through genetic inheritance?
- Are the health benefits of biking as a form of transportation cancelled out or decreased as a result of bike accidents?
- Do concrete sidewalks and paved roads increase the incidence of osteoarthritis?
- Do bad haircuts have a significant reduction in mental health/sense of self worth?
- Does fasting increase general tolerance to being uncomfortable?
- Does being able to freestyle rap improve general ability to articulate yourself and conversate with others?

**Reply by ADParedes** [reply]
- should food start to be considered a drug, where bad food can be considered as deleterious as smoking a cigarette? 
- does cheap cat and dog food increase the incidence of mouth cancer? 
- can a frequency of targeted light promote therapy, in particular, healing in chronic wound healing settings? 
- does targeted and light promote ATP generation?
- does light provide "boost" white blood cells to accelerate the inflammatory phase and promote a pro-regenerative phase?
- Can"striving" for excellence and perfectionism lead to clinical psychiatric diagnosed issues?
- Can the use of candles and fire light, in lieu of electrically generated light sources, provide any health benefits
